data/dfndata/npc/mounts.dfn

```diff
@@ -810,4 +786,23 @@ NAME=an ethereal tiger
 ID=0x04e7
 SKIN=0x4001
 ORIGIN=hs
+}
+
+//hiryu - Base NPC with shared DFN tags
+[base_hiryu]
+{
+GET=base_mounts
+NAME=#//hiryu
+ID=0x0f3
+ORIGIN=se
+}
+
+[hiryu]
+{
+GETSE=hiryu_se
+}
+
+[lesserhiryu]
+{
+GETSE=lesser_hiryu_se
```

Comment:
Add one of these for each expansion, so adding NPC with wrong expansion selected doesn't break the server. We can update where these point at when we have definitions for different eras:

GETUO=lesser_hiryu_se
GETT2A=lesser_hiryu_se
GETUOR=lesser_hiryu_se
GETTD=lesser_hiryu_se
GETLBR=lesser_hiryu_se
GETAOS=lesser_hiryu_se
GETSE=lesser_hiryu_se
GETML=lesser_hiryu_se
GETSA=lesser_hiryu_se
GETHS=lesser_hiryu_se
GETTOL=lesser_hiryu_se